advanced/firmware/.cargo/config

```diff
@@ -1,8 +1,8 @@
-[target.thumbv7em-none-eabi]
+[target.thumbv7em-none-eabihf]
```

Comment:
I think this could also be:

```suggestion
[target.'cfg(all(target_arch = "arm"', target_os = "none"))']
```

That way you can select *any* supported thumbvX target and it would just work.

Reply:
ah, but then it'd be out of sync with our blogpost which uses this as an example: https://ferrous-systems.com/blog/run-rust-on-your-embedded-device-from-vscode/

My stance would be: the way it is now is easier to parse for someone who is learning a lot of new concepts already. We don't need to "future-proof" here, and if someone is wondering how to generalize this line they can either ask or already have enough mental space for new concepts to figure it out themselves

Reply:
Ah, that makes sense when someone looks at this, yeah.